tests/unit/components/app-auto-complete/AppAutoComplete.spec.jsx

```diff
@@ -37,3 +39,93 @@ describe('AppAutoComplete test', () => {
     expect(onChange).toHaveBeenCalled()
   })
 })
+describe('AppAutoComplete test', () => {
+  beforeEach(() => {
+    renderWithProviders(
+      <AppAutoComplete
+        onChange={onChange}
+        options={optionsCity}
+        sx={styles}
+        textFieldProps={{
+          label: labelCity
+        }}
+        type='text'
+        value={value}
+      />
+    )
+  })
+  it('Should render Autocomplete and open dropdown when clicking on City field', async () => {
+    const autocomplete = screen.getByLabelText(/common.labels.city/i)
+    fireEvent.mouseDown(autocomplete)
+
+    await waitFor(() => {
+      expect(screen.getByRole('listbox')).toBeInTheDocument()
+    })
+
+    const option = screen.getByText('Verhovyna')
+    fireEvent.click(option)
+
+    expect(onChange).toHaveBeenCalled()
+  })
+  it('Should display dropdown list in alphabetic order and show text cursor when clicking on City field', async () => {
+    const cityField = screen.getByRole('combobox')
+    fireEvent.mouseDown(cityField)
+    fireEvent.focus(cityField)
+    fireEvent.click(cityField)
+
+    await waitFor(() => {
+      expect(cityField).toHaveFocus()
+    })
+    
+    await waitFor(() => {
+      expect(screen.getByRole('listbox')).toBeInTheDocument()
+    })
+
+    const optionsInDropdown = screen.getAllByRole('option')
+    const cityNames = optionsInDropdown.map(option => option.textContent)
+    const sortedCityNames = [...cityNames].sort()    
```

Comment:
Strange.. You're testing component sort options, but you are sorting by yourself. Is this test showing something or I didn't get it?